Game/Resources/lowpoly_bush.json

```diff
@@ -0,0 +1,75 @@
+{
+	"children" : 
+	[
+		{
+			"DirectionalLight" : 
```

Comment:
Won't this insert alot of directional lights into the game when this scene is instantiated?

Reply:
Correct. Forgot to clean this up.